src/pages/index.jsx

```diff
@@ -2,32 +2,27 @@ import Link from "next/link";
 import { useTranslation } from "next-i18next";
 import { serverSideTranslations } from "next-i18next/serverSideTranslations";
 import * as React from "react";
-import Footer from "@/components/Footer";
+
+import HomePageWrapper from "@/components/Home/HomePageWrapper";
+
 import Layout from "@/layout/Layout";
 
 export default function HomePage() {
     const { t } = useTranslation("common");
 
     return (
-        <>
-            <Layout>
-                <p>{t("test")}</p>
-                <div
-                    style={{
-                        display: "flex",
-                        flexDirection: "row",
-                        gap: "20px",
-                    }}
-                >
-                    <Link href='/' locale='en'>
-                        English
-                    </Link>
-                    <Link href='/' locale='ar'>
-                        العربية
-                    </Link>
-                </div>
-            </Layout>
-        </>
+        <Layout>
+            <p>{t("test")}</p>
+            <div style={{ display: "flex", flexDirection: "row", gap: "20px" }}>
+                <Link href='/' locale='en'>
+                    English
+                </Link>
+                <Link href='/' locale='ar'>
+                    العربية
+                </Link>
+            </div>
```

Comment:
Completely forgot about it ... we need to remove it @M07am3dM

Reply:
I comment it, what about the typo in the tickets @dzc0d3r .